### PR TITLE
(Blog) 2021-02-12 Status Update

### PR DESCRIPTION
**Pull request by david22swan**

### CODE REVIEW

File: _posts/2021-02-12-status-update.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+Check [all the ways to reach us]({% link _posts/2021-01-20-reaching-out.md %}) if you want to directly contact us about anything module related.
+
+## New Module / Gem Releases
+
+The following modules were released this week:
+
+- [`puppetlabs-firewall`][puppetlabs-firewall] (`2.8.1`)
+- [`puppetlabs-postgresql`][puppetlabs-postgresql] (`6.10.1`)
+- [`puppetlabs-puppet_litmus`][puppetlabs-puppet_litmus] (`0.23.1`)
```
Comment (DavidS) [inline _posts/2021-02-12-status-update.md:31]: `puppetlabs-` prefix is incorrect.

File: _posts/2021-02-12-status-update.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+- [`puppetlabs-puppetlabs_spec_helper`][puppetlabs-puppetlabs_spec_helper] (`3.0.0`)
```
Comment (DavidS) [inline _posts/2021-02-12-status-update.md:32]: same here.

**Review by DavidS — COMMENTED** [review]
- [x] mention IAC-1008, last(?) module with inappropriate language removed? reason to celebrate!
- [ ] https://github.com/puppetlabs/puppetlabs-stdlib/pull/1161 was merged, fixing a nasty memory/behavior change issue on stdlib on ruby 2.7
- [x] missing shoutout for ekohl and vchepkov on their help with nasty postgresql/systemd module upgrade issue
- [ ] @michaeltlombardi @pmcmaw something something dsc first milestone link blog posts again?

File: _posts/2021-02-12-status-update.md (hunk 2 above)
Comment (david22swan) [reply _posts/2021-02-12-status-update.md:32]: Fixed

File: _posts/2021-02-12-status-update.md (hunk 1 above)
Comment (david22swan) [reply _posts/2021-02-12-status-update.md:31]: Fixed

**Comment by david22swan** [comment]
Blog has been updated with changes

File: _posts/2021-02-12-status-update.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+  [puppet-litmus-pr-385]:       https://github.com/puppetlabs/puppet_litmus/pull/385
+  [puppetlabs-testing-issues]:  https://github.com/puppetlabs/puppetlabs-testing/runs/1863891582?check_suite_focus=true
+
+## Removal of Inappropriate Terminology
+
+As of the linked [ticket][puppetlabs-helm-pr-140] being merged in puppetlabs-helm I am happy to announce that we have finished our work in the removal of all possible instances of inappropriate or offensive terminologies from our supported modules and altered their primary branches to be the `main` branch.
+As part of this announcement I would like to remind everyone the the old `master` is no longer updated and will be removed from the modules in the future and so would urge anyone directly referencing it to instead target the `main` branch.
+
+  [puppetlabs-helm-pr-140][https://github.com/puppetlabs/puppetlabs-helm/pull/140]
```
Comment (sanfrancrisko) [inline _posts/2021-02-12-status-update.md:104]: Incorrect link format:
```suggestion
  [puppetlabs-helm-pr-140](https://github.com/puppetlabs/puppetlabs-helm/pull/140)
```

File: _posts/2021-02-12-status-update.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+## A Sad Day for the IAC Team
+
+Alongside all this weeks good news we have some sad news to announce. It is an end of an era, a good friend and colleague [TP Honey][TP] is leaving Puppet today for pastures new. TP has been a role model and mentor, to all within the IAC team and his contributions have helped so many in our opensource community. As he leaves the team would like to remind TP that all PR's are welcome.
```
Comment (sanfrancrisko) [inline _posts/2021-02-12-status-update.md:108]:
```suggestion
Alongside all this weeks good news we have some sad news to announce. It is an end of an era, a good friend and colleague [TP Honey][TP] is leaving Puppet today for pastures new. TP has been a role model and mentor, to all within the IAC team and his contributions have helped so many in our Open Source Community. As he leaves the team would like to remind TP that all PRs are welcome.
```

**Review by sanfrancrisko — COMMENTED** [review]
Just minor tweaks.

I like the idea of having the links closer to the section that is using them 👍

File: _posts/2021-02-12-status-update.md (hunk 3 above)
Comment (david22swan) [reply _posts/2021-02-12-status-update.md]: Link fixed

File: _posts/2021-02-12-status-update.md (hunk 4 above)
Comment (david22swan) [reply _posts/2021-02-12-status-update.md]: Grammar updated

**Review by DavidS — APPROVED** [review]

**Pull request merged**